tests/database/Db/Adapter/Pdo/DescribeColumnsCest.php

```diff
@@ -56,4 +56,29 @@ public function dbAdapterPdoDescribeColumnsOnUpdate(DatabaseTester $I)
         $I->assertSame('CURRENT_TIMESTAMP on update CURRENT_TIMESTAMP', $columns[2]->getDefault());
         $I->assertSame('NULL on update CURRENT_TIMESTAMP', $columns[3]->getDefault());
     }
+
+    /**
+     * Tests Phalcon\Db\Adapter\Pdo :: describeColumns()
+     *
+     * @author Jeremy PASTOURET <https://github.com/jenovateurs>
+     * @since  2019-12-07
```

Comment:
Date is wrong :)

Reply:
You're right, thanks for your comments and for your review ;-).